about.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style/index.css" />
+  </head>
+  <body>
+    <section class="container-md d-flex flex-column align-items-center my-5">
+      <p class="h1 heading pt-3 pt-md-0 text-center mt-4 mb-1">Let's play Chess</p>
+      <h1
+        class="about-header display-2 fw-bolder text-uppercase text-center mb-4"
+      >
+        Jah Malet Fetari International Chess Tournament 2023
+      </h1>
+      <div class="sub-heading text-center mb-2 mt-4 mx-2">
+        <p class="fs-5 p-4 p-md-5">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat
+          dignissimos dolore recusandae consectetur iste, sapiente eaque harum
+          impedit dolores corrupti ab minus quis, molestiae excepturi sit cumque
+          tempora iure. Explicabo impedit dolores magnam dicta unde ea eius
+          error! Dolorem similique fuga molestiae eveniet atque praesentium
+          doloribus libero nobis totam tempore.
+        </p>
```

Comment:
- [ ] Please as pointed earlier, kindly ensure that the background image and the paragraph styling is according to the behance design. It will be of great advantage to us if we pay attention to details as developers. ℹ️